static/js/controllers/inbox.js

```diff
@@ -520,7 +520,7 @@ var feedback = require('../modules/feedback'),
             .tooltip({
               placement: 'bottom',
               trigger: 'manual',
-              container: $(this).closest('.inbox-items, .item-content')
+              container: $(this).closest('.inbox-items, .item-content, .page')
```

Comment:
Also, was this what actually fixed it? That we had some tooltip logic, and it needs one of these classes to bind it's hidden html to?

Reply:
Yes this is what fixed it. You have to opt in to bootstrap tooltips. If you don't specify the `container` then it defaults to `body` which doesn't work in our UI which floats around all over the place.